W 12228439 exchange add support for json fragments jh (#231)
mulesoft/docs-exchange: modules/ROOT/pages/to-create-an-asset.adoc

* For OAS, RAML, RAML fragments, AsyncAPI, HTTP, WSDL, rulesets, and custom assets, create the asset directly using the Exchange *Publish new asset* menu.
* For an example or template, create each using the Mavenize feature in Anypoint Studio, and publish each to Exchange.
* For a connector, policy, example, or template, see xref:to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven].

> * You can also publish REST-APIs and API Spec Fragments using Design Center and publish the API to Exchange.

Reviewer: ```suggestion
* You can also create REST APIs and API Spec Fragments using Design Center and publish the API to Exchange.
```

> * You can also publish a RAML, OAS components, and JSON and RAML fragments using Design Center and publish the API to Exchange.

Reviewer 2: What do you think if we talk about asset types instead of different technologies? I think it can avoid misunderstandings and it is more concise

```suggestion
* You can also publish REST-APIs and API Spec Fragments using Design Center and publish the API to Exchange.
```

Reviewer 3: yes this makes sense to me, if we explain the granularity within REST-APIs below

Author: fixed

. In Exchange, select *Publish new asset*.
. Enter the asset portal name.
. Select the asset type from the drop-down list:

* REST API - RAML: Provide a RAML API specification file. RAML specifications must be a RAML file (.raml).
* REST API - OAS: Provide an OAS API specification file. OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
* SOAP API - WSDL: Provide a WSDL API specification file. SOAP specifications file can be either a WSDL (.wsdl) or XML (.xml) file.
* AsyncAPI: Provide an AsyncAPI specification file in a YAML (.yaml) file.

> * API Spec Fragment:
> +
> RAML: Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml). 
> +
> JSON: Provide an API Fragment JSON specification (.json) file. 
> +
> OpenAPI Specification (OAS): Provide an API Fragment OpenAPI specification file either JSON or YAML.

Reviewer 2: ```suggestion
* API Spec Fragment - RAML: Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml). 
* API Spec Fragment - JSON: Provide an API Fragment JSON specification (.json) file. 
* API Spec Fragment - OpenAPI Specification (OAS): Provide an API Fragment OpenAPI specification file either JSON or YAML.
```

Reviewer 2: To maintain consistency with the text above we could use the same structure.

Reviewer 3: agreed
